wolfcrypt/src/asn.c

```diff
@@ -955,7 +957,15 @@ int GetInt(mp_int* mpi, const byte* input, word32* inOutIdx, word32 maxIdx)
         return ASN_GETINT_E;
     }
 
+#ifdef HAVE_WOLF_BIGINT
```

Comment:
After looking at wc_DhGenerateKeyPair_Async, I'm not sure it make sense to set the raw here.